scripts/common/actor.js

```diff
@@ -195,7 +218,27 @@ export class AgeOfSigmarActor extends Actor {
         this.power.capacity += item.power.capacity;
     }
 
-    _computeAttack() {
+    _sizeToken(isSwarm) {
+        if(isSwarm) return; //Swarms are variable let the GM decide Size
+
+        let size = this.bio.size; 
+
+        if(size <= 2) {
+            this.data.token.width = 1;
+            this.data.token.height = 1;
+        } else if(size === 3) {
+            this.data.token.width = 2;
+            this.data.token.height = 2;
+        } else if(size === 4) {
+            this.data.token.width = 3;
+            this.data.token.height = 3;
+        } else if(size === 5) {
+            this.data.token.width = 4;
+            this.data.token.height = 4;
+        }
+    }
```

Comment:
Does this work in dynamically sizing active tokens? It doesn't seem like it would, that would be desirable right?

Reply:
No Intention to do that you set it once and it would never ever change for any reason nothing gets bigger or shrinks. Dynamically Sizing something on the canvas is unnecessary in my book.

Reply:
So the only issue I see with this is that synthetic actors need to support dynamic sizing if the user wants an individual to be a different size, otherwise it's rather awkward for them to change specific tokens. I'm not sure how often that might be relevant though.

Reply:
Well the Option to toggle it on or off specifically tells the user it won't work for anything on the already on the canvas. If people use the exact token size rules these values are based on they wouldn't change an actors size during play unless it was user error in the during set up when the GM created the NPC.